yarn.lock

```diff
@@ -3221,7 +3221,7 @@ gulplog@^1.0.0:
   dependencies:
     glogg "^1.0.0"
 
-handlebars-utils@^1.0.4, handlebars-utils@^1.0.6:
+handlebars-utils@^1.0.4:
```

Comment:
Should this not entirely be gone now?

Reply:
Due the maze of depenendencies, this is still used for a dev package called "verb". This should probably not be needed, but given that is a dev dependency we should be good for now